frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -239,6 +239,13 @@ private void saveAttachmentUsages(ResourceRequest request, ResourceResponse resp
 
     }
 
+    private void setProjectPathToAttachmentUsages(List<AttachmentUsage> usagesToCreate, Project project) {
+        usagesToCreate.forEach(usage -> {
+            if (!usage.usageData.getLicenseInfo().isSetProjectPath())
+                usage.usageData.getLicenseInfo().setProjectPath(project.getId());
```

Comment:
May be we can use `usage.getUsageData()` here.

Reply:
Refactored the code as per review comment.